template/DEVELOPING/environment-variables.md

```diff
@@ -1,11 +1,54 @@
 # Environment Variables
 
-We use
-[@nest/env](https://github.com/vercel/next.js/blob/master/docs/basic-features/environment-variables.md#loading-environment-variables)
-to load environment variables from `.env*` files in both the frontend and
-backend packages. See the @nest/env readme for detailed documentation.
+## Reading Environment Variables
 
-Briefly you can have multiple env files that are loaded depending
+All of the environment variables that the app reads are declared in [Env][]. We
```

Comment:
Clearer to spell out the file path for the link, so we don't need to click on it to know where it is.